packages/caravan-fees/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import BigNumber from "bignumber.js";
+
+export const DEFAULT_RBF_SEQUENCE = 0xfffffffd;
+export const MIN_FEE_RATE = 1;
+export const MAX_FEE_RATE = 5000;
+export const DEFAULT_FEE_INCREASE_THRESHOLD = 1.1;
+export const MIN_DUST_AMOUNT = new BigNumber(546);
```

Comment:
I think exported values and constants should probably be strings that way any library that consumes this can be BigNumber library agnostic.

Reply:
It was addressed earlier in the commit [Refactor fee bumping package for improved functionality and integration](https://github.com/caravan-bitcoin/caravan/commit/b7b2fa5ad6851ad07bb3087d251a6b3191b71a80)